<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VWebProxyWidget</class>
 <widget class="QWidget" name="VWebProxyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInternal">
      <attribute name="title">
       <string>Internal</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="glInternal">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabHttpServer">
           <attribute name="title">
            <string>HTTP Server</string>
           </attribute>
           <widget class="QWidget" name="gridLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>160</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="glHttpServer"/>
           </widget>
          </widget>
          <widget class="QWidget" name="tabHttpsServer">
           <attribute name="title">
            <string>HTTPS Server</string>
           </attribute>
           <widget class="QWidget" name="gridLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>160</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="glHttpsServer"/>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabExternal">
      <attribute name="title">
       <string>External</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="glExternal"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tabDataChange">
      <attribute name="title">
       <string>DataChange</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="glDataChange">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabInbound">
           <attribute name="title">
            <string>Inbound</string>
           </attribute>
           <widget class="QWidget" name="gridLayoutWidget_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>160</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="glInbound"/>
           </widget>
          </widget>
          <widget class="QWidget" name="tabOutbound">
           <attribute name="title">
            <string>Outbound</string>
           </attribute>
           <widget class="QWidget" name="gridLayoutWidget_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>160</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="glOutbound"/>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabOther">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="glOther"/>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
